README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Personal Profile
+## Name: Xumeng Yan
+Aspect | Information
+------------ | -------------
+**Pronoun** | She/Her/Hers
+**Program** | MAPSS
+**Major** | Sociology
+**Research Interest** | *Social Stratification, Social Movements*
+
+* A couple things I enjoy:
+  * Experimenting with cooking recipes
+  * Chasing my pets
+  * Lying on the floor and silently listening to music
+
+* Favorite lyrics:
+> *If you want to make the world a better place*
+> *Take a look at yourself, and then make a change* (Michael Jackson)
+
+![Michael Jackson](Michael_Jackson_-_Man_in_the_Mirror.png)
+
+* You are welcome to add me on Facebook (~~although I don't talk a lot~~)! :point_down:
```

Comment:
I also used emojis for my bio. One amazing thing about programming is that the more you go further the more you learn. In this sense, emojis in your code is a plus!